process/block/metablock.go

```diff
@@ -2032,7 +2032,10 @@ func (mp *metaProcessor) createShardInfo() ([]data.ShardDataHandler, error) {
 		shardData.PrevRandSeed = shardHdr.GetPrevRandSeed()
 		shardData.PubKeysBitmap = shardHdr.GetPubKeysBitmap()
 		if mp.enableEpochsHandler.IsFlagEnabledInEpoch(common.EquivalentMessagesFlag, shardHdr.GetEpoch()) {
-			_, shardData.PubKeysBitmap = shardHdr.GetPreviousAggregatedSignatureAndBitmap()
+			proof := shardHdr.GetPreviousProof()
```

Comment:
The data for a notarized shard header should be related to that shard header not the previous.
here the proof for the header should be used not the one included in the header, which is for the previous one.

Also